manifests/pip/bootstrap.pp

```diff
@@ -21,24 +22,26 @@
     }
     if $version == 'pip3' {
       exec { 'bootstrap pip3':
-        command => '/usr/bin/curl https://bootstrap.pypa.io/get-pip.py | python3',
-        unless  => 'which pip3',
-        path    => $python::params::pip_lookup_path,
-        require => Package['python3'],
+        command     => '/usr/bin/curl https://bootstrap.pypa.io/get-pip.py | python3',
+        environment => [ "HTTP_PROXY=${http_proxy}", "HTTPS_PROXY=${http_proxy}", "http_proxy=${http_proxy}", "https_proxy=${http_proxy}" ],
```

Comment:
why set it as uppercase and lowercase environment variables?

Reply:
AIX curl needed it to be lower case.